LifeSpace/Debug/LogViewer.swift

```diff
@@ -0,0 +1,112 @@
+//
+//  LogShareView.swift
+//  LifeSpace
+//
+//  Created by Vishnu Ravi on 11/8/24.
+//
+
+import OSLog
+import Spezi
+import SwiftUI
+
+
+struct LogViewer: View {
+    @Environment(LogManager.self) var manager
+    
+    @State private var startDate: Date = Calendar.current.date(byAdding: .day, value: -1, to: Date()) ?? Date()
+    @State private var endDate = Date()
+    @State private var selectedLogType: LogType = .all
+    @State private var logs = ""
+    @State private var isLoading = false
+    
+    var body: some View {
+        VStack {
+            /// Date range selection
+            HStack {
```

Comment:
Might make sense to display the date pickers in a VStack, the spacing between From <-> Picker and To <-> Picker looks a bit strange.

Reply:
Updated this to show the pickers in a VStack, and also include the time as well as the date for filtering.